app/components/StartStreamingButton.vue

```diff
@@ -1,7 +1,11 @@
 <template>
 <button
   class="button button--go-live"
-  :class="{'button--soft-warning': (streamingService.isStreaming && streamingService.stopStreaming), 'button--action': streamingService.startStreaming}"
+  :class="{
+    'button--soft-warning': (streamingService.isStreaming && streamingService.stopStreaming),
```

Comment:
`streamingService.stopStreaming` actually is a method, do you know that is purpose of that in the button class definition?

Reply:
Huh, you're right.  That's super weird.  I have no idea where this logic came from, but it's pointless.  I will clean it up.